src/components/cc-pricing-product-consumption/cc-pricing-product-consumption.types.d.ts

```diff
@@ -0,0 +1,17 @@
+import { PricingSection } from '../common.types.js';
+
+interface PricingProductConsumptionStateLoading {
+    state: 'loading';
+}
+
+interface PricingProductConsumptionStateError {
+    state: 'error';
+}
+
+interface PricingProductConsumptionStateLoaded {
+    state: 'loaded';
+    name: string;
+    sectionList: PricingSection[];
```

Comment:
We often use plural but we decided to use the `List` suffix for `currencyList` and `temporalityList` to avoid confusion with `currency` and `temporality` (which we have also renamed to `selectedCurrency` + `selectedTemporality`).

Considering this, I think we should use the same logic, at least inside all the pricing components.
Since we've renamed `currency` and `temporality`, we could use plurals for lists to be consistent with components other than pricing components.

I'm ok with both solutions but I guess the last option (plurals) is better for consistency, WDYT?